_posts/2018-12-24-ApiaryUI.md

```diff
@@ -0,0 +1,131 @@
+---
+title: ApiaryUI - Interactive Documentation v4
+excerpt: Introduction version 4 of Apiary Interface Documentation
+layout: post
+date: 2018-12-24 12:00:00 +0200
+author: char0n
+published: true
+comments: true
+---
+
+After months of hard work and burning the midnight oil I am very proud to announce
+the **ApiaryUI** - brand new, fully reviewed and refactored version of 
+[Apiary Interactive Documentation v3](https://help.apiary.io/tools/interactive-documentation/) (v3).
+
+ApiaryUI has already been deployed to [apiary.io](https://apiary.io/) on 20th of December 2018. 
+ApiaryUI is currently in what we call EAP (Early Access Program).
+Although it's already GA (Generally Available), it’s not yet fully production ready.
+We want to share it with the world now and get the invaluable feedback from you - Apiary users.
+
+ApiaryUI is implemented in a way, that allows complete separation from [apiary.io](https://apiary.io/)
+and possibly forming a new separate product in the future. Some features from v3 are still missing
```

Comment:
```suggestion
 and possibly forming a new separate product in the future. Some features from our existing documentation product are still missing
```

I’m wondering if it makes sense to avoid the term “v3”, I am not sure that many of our customers or readers of this post will know what “v3” is exactly.

Reply:
"Some features" spreads FUD. I'd specify which at the end of article and say whether we do plan to implement them or not.